requirements/base.txt

```diff
@@ -2,48 +2,48 @@
 # like Pillow and psycopg2
 # See http://bitly.com/wheel-building-fails-CPython-35
 # Verified bug on Python 3.5.1
-wheel==0.34.2
-argon2-cffi==19.2.0
-asn1crypto==1.3.0
+wheel==0.35.1
+argon2-cffi==20.1.0
+asn1crypto==1.4.0
 awesome-slugify==1.6.5
-beautifulsoup4==4.9.0
-boto3==1.12.37
-botocore==1.15.37
-certifi==2020.4.5.1
-cffi==1.14.0
+beautifulsoup4==4.9.2
+boto3==1.15.6
+botocore==1.18.6
+certifi==2020.6.20
+cffi==1.14.3
 chardet==3.0.4
-click==7.1.1
-cryptography==2.9
+click==7.1.2
+cryptography==3.1.1
 defusedxml==0.6.0
-Django==2.2.11
-django-allauth==0.41.0
+Django==2.2.13
```

Comment:
2.2.16 is the latest version of Django 2.2

Reply:
https://github.com/SFDO-Tooling/sfdoc/pull/313/files#diff-7fe11226cff646f5d9f35faa76217059R18